gorm/README.md

```diff
@@ -0,0 +1,40 @@
+# Go2Sky with Gorm
```

Comment:
I think this doc should be linked from https://github.com/SkyAPM/go2sky-plugins#trace-plugins